timeline-for-beaver-builder-module/includes/frontend.php

```diff
@@ -77,12 +78,12 @@
 					</div>
 
 					<!--Timline-description-->
-					<div class="bb-timline-dec"><?php echo $timeline1->timeline_editor; ?></div>
+					<div class="bb-timline-dec"><?php echo wp_kses_post( $timeline1->timeline_editor ); ?></div>
```

Comment:
Make sure this is tested

Reply:
If we are providing editor which allows HTML  too it should work properly, backward comp needs to be tested